UI/HomeIdeasUI/Directory.Packages.props

```diff
@@ -0,0 +1,25 @@
+<Project ToolsVersion="15.0">
+  <ItemGroup>
```

Comment:
We are inconsistent whether we use two spaces or a tab for XML props/targets/csproj files.

This doesn't block merging this PR, but something we should unify.